platform/email/platform-email-api/src/main/java/org/codice/ddf/platform/email/api/EmailService.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.email.api;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+
+/**
+ * Provides a light-weight interface to javax.mail. The user should call {@link #createSession()}
+ * to get a session that is preconfigured with the hostname and port number of the email server.
+ * Next, construct a {@link Message} and submit it to {@link #send(Message)}.
+ *
+ * Example:
+ * <pre>
+ * {@code
+ *
+ * EmailService emailService = <get an instance>
+ *
+ * Session session = emailService.createSession();
+ *
+ * MimeMessage mimeMessage = new MimeMessage(session);
+ * mimeMessage.setFrom(new InternetAddress("from@test.com"));
+ * mimeMessage.addRecipient(Message.RecipientType.TO, new InternetAddress("to@test.com"));
+ * mimeMessage.setSubject("The Subject Line");
+ *
+ * BodyPart messageBodyPart = new MimeBodyPart();
+ * messageBodyPart.setText("The Body Text");
+ *
+ * Multipart multipart = new MimeMultipart();
+ * multipart.addBodyPart(messageBodyPart);
+ *
+ * mimeMessage.setContent(multipart);
+ *
+ * emailService.send(mimeMessage);
+ *
+ * }
+ * </pre>
+ */
+public interface EmailService {
```

Comment:
I think we should include the experimental header since this is a new interface.